src/resourceClient/apiUrlBuilder/searchUrl.ts

```diff
@@ -2,7 +2,7 @@ import { Options } from '../resourceClient';
 import ApiUrlBuilder from './apiUrlBuilder';
 
 export default class SearchUrl extends ApiUrlBuilder {
-  // @ts-expect-error
+  // eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
```

Comment:
unused vars can be marked with underscore and eslint won't clarify them as error then

Reply:
<img width="620" alt="image" src="https://github.com/statsfm/applemusic.js/assets/27011453/412643b8-8a81-4f9c-99e2-35ecacc8db88">